Description: Don't use FindFLTK 1.1 to find FLTK 1.3
 CMake's FindFLTK is for FLTK 1.1, and is not recommended for FLTK1.3
 (see fltk1.3's README.CMake.txt): it links FLTK statically,
 and links against several things libfltk1.3-dev doesn't depend on
 (libxft-dev,libxinerama-dev,libfontconfig1-dev,libjpeg-dev,libpng-dev)
Author: Rebecca Palmer
Forwarded: https://gitorious.org/fg/fgrun/merge_requests/8

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,7 +128,10 @@
 
 find_package(Boost REQUIRED)
 find_package(ZLIB REQUIRED)
-find_package(FLTK REQUIRED)
+find_package(FLTK REQUIRED NO_MODULE)
+include(${FLTK_USE_FILE})
+#Not sure if we actually need them all
+set(FLTK_LIBRARIES "fltk_forms_SHARED;fltk_gl_SHARED;fltk_images_SHARED;fltk_SHARED")
 find_package(OpenSceneGraph 2.8.1 REQUIRED osgText osgDB osgParticle osgUtil osgViewer osgGA)
 find_package(SimGear 2.8.0 REQUIRED)
 if (ENABLE_NLS)
